utils/serverStatistic.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const { default: fetch } = require("node-fetch");
+
+const application_id = 'd78e7f67147305f3042bef05755fa168';
+const MAX_ACCOUNTS_PER_REQUEST = 100;
+const TOTAL_ACCOUNTS = 594859325;
+const BASE_URL = 'https://api.wotblitz.eu/wotb';
+const allPlayerStats = { account: {}, tanks: [] };
+
+const statsNames = ['battles', 'damage_dealt', 'damage_received', 'frags', 'spotted', 'wins', 'losses', 'survived_battles'];
+const achievementsName = ['markOfMastery', 'markOfMasteryI', 'markOfMasteryII', 'markOfMasteryIII'];
+const getStatsObject = (statsName, stats) => Object.fromEntries(statsName.map((statName) => [statName, stats[statName] || 0]));
+
+const getInfo = async (url) => {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data.data;
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
+const getVehicleStats = async (accountId) => {
+  const result = {};
```

Comment:
Removed the unnecessary result object and directly used an array to store the mapped tank statistics.
```suggestion
```